src/components/DeleteModal.vue

```diff
@@ -0,0 +1,197 @@
+<template>
```

Comment:
@tupic98 as far as I can see, we can make a Component (GlobalModal) and inject the body based on delete or detail. With that, we can achieve DRY